Penumbra/UI/AdvancedWindow/ModEditWindow.Models.cs

```diff
@@ -508,6 +508,11 @@ private static bool DrawOtherModelDetails(MdlFile file, bool _)
                 ImGuiUtil.DrawTableColumn(file.VertexDeclarations.Length.ToString());
                 ImGuiUtil.DrawTableColumn("Stack Size");
                 ImGuiUtil.DrawTableColumn(file.StackSize.ToString());
+                for (var lod = 0; lod < file.Lods.Length; lod++)
+                {
+                    ImGuiUtil.DrawTableColumn("LoD " + lod + " Triangle Count");
```

Comment:
Imo, never use more than one `+` for string concatenation, use interpolation instead. No idea if the compiler optimizes away the intermediary string, but eh, that's what interpolation is for anyway.

Reply:
Looks like you already beat me to it, but I'll keep that in mind for the future.

My expectation would be that the compiler would optimize the concatenation in this case to a call to `string.concat()`. As I've just learned, compiler optimization for interpolated strings in C# is [a bit complex](https://devblogs.microsoft.com/dotnet/string-interpolation-in-c-10-and-net-6/), so I haven't a clue how concat vs. interpolation would compare in terms of performance for a simple example like this.